app/views/welcome/_map.html.slim

```diff
@@ -42,35 +41,30 @@ css:
 
 javascript:
     // initialize the map
-    var map = L.map('map', { zoomControl: #{defined?(zoomControl) ? zoomControl : true } }).setView([52.511946, 13.406166], 10);
-    var imgNotAvailable = 'https://www.postfreeadvertising.com/wp-content/uploads/2018/08/no-image-available-grid-263x172.png'
-    var toBeUsedTile = 'https://api.mapbox.com/styles/v1/mapbox/light-v10/tiles/{z}/{x}/{y}?access_token={accessToken}'
-    var stadiamaps = 'https://tiles.stadiamaps.com/tiles/alidade_smooth/{z}/{x}/{y}.png'
-    var osmMap = 'https://maps.wikimedia.org/osm-intl/{z}/{x}/{y}.png'
-    var cartoDB = 'https://{s}.basemaps.cartocdn.com/light_all/{z}/{x}/{y}{r}.png'
+    var map = L.map('map', { zoomControl: #{defined?(zoomControl) ? zoomControl : true } }).setView([52.511946, 13.406166], 12);
+    var myStyle = 'https://api.mapbox.com/styles/v1/myusernameisthis/ck8k8w25g02d01jmnnbehc46d/tiles/512/{z}/{x}/{y}?access_token={accessToken}'
     // load a tile layer
     L.tileLayer(
-    toBeUsedTile,
+    myStyle,
     {
         attribution: 'Map data &copy; <a href="https://www.openstreetmap.org/">OpenStreetMap</a> contributors, <a href="https://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>, Imagery © <a href="https://www.mapbox.com/">Mapbox</a>',
         maxZoom: 16,
         minZoom: 3,
-        //accessToken: 'pk.eyJ1IjoibXl1c2VybmFtZWlzdGhpcyIsImEiOiJjazhndmhzNmYwNHV3M3NzMDcwNGs1ZjZzIn0.2ZRug8MHanm-5yQd9ptfGg',
-        accessToken: 'pk.eyJ1IjoiaGVzc2xhdSIsImEiOiJjazhpcmFxbmwwMmR5M2hwYWptbGdsbXNuIn0.uvVb2KclwLDJGMbr_PgXaQ',
+        accessToken: 'pk.eyJ1IjoibXl1c2VybmFtZWlzdGhpcyIsImEiOiJjazhndmhzNmYwNHV3M3NzMDcwNGs1ZjZzIn0.2ZRug8MHanm-5yQd9ptfGg',
```

Comment:
is this a "secret" token? if so we should change it and stop sharing it on github =D

Reply:
No, it is restricted to the urls map.nachbarschaftsmarktplatz.de and shop.nachbarschaftsmarktplatz.de

Reply:
could you also add staging.nachbarschaftsmarktplatz to the list of supported domains?